File: sdk/php/src/Resources/ServiceStatus.php
<?php

namespace HiTechCloud\SDK\Resources;

class ServiceStatus extends BaseResource
{
    /**
     * List statuses
     */
    public function list(array $params = []): array
    {
        return $this->http->get('/api/statuses', $params);
    }

    /**
     * Update status
     */
    public function update(int $statusId, array $data): array
    {
        return $this->http->put('/api/statuses/' . $serviceId, $data);
    }

}

File: sdk/csharp/Resources/ServiceStatus.cs
using System.Collections.Generic;
using System.Threading.Tasks;

namespace HiTechCloud.SDK.Resources;

public class ServiceStatus : BaseResource
{
    public ServiceStatus(HttpClientWrapper http) : base(http) { }

    public async Task<Dictionary<string, object?>> StatusesAsync()
    {
        return await Http.GetAsync("/api/statuses");
    }

    public async Task<Dictionary<string, object?>> UpdateStatusAsync(int id, Dictionary<string, object> data)
    {
        return await Http.PutAsync(string.Format("/api/statuses/{id}", id), data);
    }
}
